File: TSINJO/Chemin_court/base/MyConnection.py
import psycopg2
class MyConnection:

    @staticmethod
    def connect():
        try:
            conn = psycopg2.connect(
                host="localhost",
                port="5432", 
                dbname="reseau", 
                user="postgres", 
                password="postgres",
            )
            return conn
        except(Exception,psycopg2.Error) as error:
            print("ERREUR", error)

File: Ruby/lalanaruby/base/MyConnection.rb
require 'pg'

class MyConnection
    def self.connect()
        # Initialize connection variables.
        host = String('localhost')
        database = String('lalanaruby')
        user = String('postgres')
        password = String('postgres')

        # Initialize connection object.
        connection = PG::Connection.new(:host => host, :user => user, :dbname => database, :port => '5432', :password => password)

    return connection
    end
end

